app/popup.tsx

```diff
@@ -31,6 +33,9 @@ const isChromeUrl = (url: string) => {
 
 export class Popup extends React.Component<POPUP_PROPS & DispatchProps, {}> {
   componentWillMount() {
+    MessageService.getEnabledStatus(this.props.tabId, (enabled: boolean) => {

```

Comment:
You are doing this anyway in the `app.tsx`. No need to do this here. 
This will just slow down and make unnecessay messaging

Reply:
Sure 👍 . Will pass the same from `app.tsx`